analyses/infercnv-consensus-cell-type/scripts/01_run-infercnv.R

```diff
@@ -193,7 +193,9 @@ sce <- sce[, !(colnames(sce) %in% duplicated_cells)]
 raw_counts_matrix <- cbind(
   as.matrix(counts(sce)),
   as.matrix(counts(ref_sce))
```

Comment:
```suggestion
  counts(sce),
  counts(ref_sce)
```
I actually don't think you need the `as.matrix` at all and then you don't need to add the lines later. When I was modifying this script for `ews-nf`, I was noticing large memory warnings so removed this and everything was fine.